chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/developer/attestation/AttestatationFormMetaData.java

```diff
@@ -0,0 +1,45 @@
+package gov.healthit.chpl.scheduler.job.developer.attestation;
+
+public class AttestatationFormMetaData {
+    private static final Long INFO_BLOCKING_CONDITION = 1L;
+    private static final Long ASSURANCES_CONDITION = 7L;
+    private static final Long ASSURANCES_CONDITION_ORIG = 2L;
+    private static final Long COMMUNICATIONS_CONDITION = 3L;
+    private static final Long API_CONDITION = 4L;
+    private static final Long RWT_CONDITION = 5L;
+
+    private static final Long NOT_APPLICABLE_RESPONSE_ID = 3L;
+    private static final Long COMPLIANT_RESPONSE_ID = 1L;
+
+    public static Long getInformationBlockingConditionId() {
+        return INFO_BLOCKING_CONDITION;
+    }
+
+    public static Long getAssurancesConditionId(Long attestationPeriodId) {
+        if (attestationPeriodId <= 3) {
```

Comment:
Is this the thing where the period ID has to be hardcoded? I thought we got everything from the attestation period -> form.